api/v1beta1/flinkcluster_types.go

```diff
@@ -206,6 +206,10 @@ type JobManagerSpec struct {
 	// Sidecar containers running alongside with the JobManager container in the
 	// pod.
 	Sidecars []corev1.Container `json:"sidecars,omitempty"`
+
+	// Defines the node affinity of the pod
+	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```

Comment:
Move this field to below NodeSelector because they are semantically close.

Reply:
All set